package.json

```diff
@@ -11,6 +11,7 @@
     "@core-ds/primitives": "^1.0.0",
     "@ifixit/localize": "^1.0.0",
     "@ifixit/toolbox": "^1.1.0",
+    "@tippy.js/react": "^3.1.1",
```

Comment:
Now this is incompatible with the version in our main repo. i.e. we'll have to have two copies of tippy. We should probably downgrade this one or upgrade the other.

Reply:
FYI, I don't mean it won't work, just that npm will now have to include both versions of tippy.

Reply:
Fixed in https://github.com/iFixit/product-page-components/pull/34/commits/571e2997b03123cfd41cd945bdef4750c2d637c4.